magento/pwa-studio
# [feature]: Provide a way to access top/intermediate level categories through left nav

## Description
This PR will resolve problem could not click to parent category in left navigation

## Related Issue





Closes #2276.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Click to hamburger button near by left logo.
2. You will see the left menu, then click to **Bottoms** item.
3. Make sure **All Bottoms** appear on the top of list submenu.

## Screenshots / Screen Captures (if appropriate)
<img width="1039" alt="Screen Shot 2020-08-08 at 3 01 05 PM" src="https://user-images.githubusercontent.com/9253798/89705497-189ddd00-d988-11ea-990d-b93e3491d802.png">


## Checklist

* I have updated the documentation accordingly, if necessary.
